docs/src/adoc/index.adoc

```diff
@@ -2072,9 +2072,114 @@ To use this module, add a Maven dependency:
 </dependency>
 ----
 
-Then configure the Jdbi factory bean in your Spring container:
+Then configure the Jdbi factory bean in your Spring container, e.g.:
 
-TODO: usage example
+[source,xml]
+----
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:aop="http://www.springframework.org/schema/aop"
+       xmlns:tx="http://www.springframework.org/schema/tx"
+       xsi:schemaLocation="
+       http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
+       http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx-2.0.xsd
+       http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop-2.0.xsd">
+
+  <!--1-->
+  <bean id="derby"
+    class="org.apache.derby.jdbc.EmbeddedDataSource"
```

Comment:
Should we consider converting the Derby stuff to h2, to be consistent with the rest of the tests?